components/ui/ShippingSimulation.tsx

```diff
@@ -0,0 +1,154 @@
+import Button from "$store/components/ui/Button.tsx";
+import Input from "$store/components/ui/Input.tsx";
+import Text from "$store/components/ui/Text.tsx";
+import { formatPrice } from "$store/sdk/format.ts";
+
+import { useCart } from "deco-sites/std/commerce/vtex/hooks/useCart.ts"
+
+import type {
+    SimulationOrderForm,
+    SKU,
+    Sla
+} from "deco-sites/std/commerce/vtex/types.ts";
+
+import { useSignal } from "@preact/signals";
+import { useCallback } from "preact/hooks";
+
+export interface Props {
+    items: Array<SKU>;
+}
+
+function ShippingContentError() {
+    return(
+        <div class="p-2">
+            <span>CEP inválido</span>
+        </div>
+    )
+}
+
+function ShippingContent({ simulation, locale, currencyCode }:{ simulation: SimulationOrderForm; locale: string; currencyCode: string }) {
+
+    if (!simulation.logisticsInfo?.length) {
+        return <ShippingContentError />;
+    }
+
+    const methods = simulation.logisticsInfo.reduce<Sla[]>((initial, logistic) => {
+          return [...initial, ...logistic.slas];
+    }, []);
+
+    if (!methods.length) {
+        return <ShippingContentError />;
+    }
+
+    const handleShippingTime = (estimate: string) => {
+        const [, time, type] = estimate.split(/(\d+)/);
+    
+        if (type === "bd") return `${time} dias úteis`;
+        if (type === "d") return `${time} dias`;
+        if (type === "h") return `${time} horas`;
+    };
+
+    return(
+        <ul class="flex flex-col gap-4 p-4 bg-hover rounded-[4px]">
+            {methods.map((method) => (
+                <li class="flex justify-between items-center border-[#e4e4e4] not-first-child:border-t-1">
+                    <Text variant="body" class="text-button text-center text-[13px]">
```

Comment:
I think it's better sticking to the design system
```suggestion
                    <Text variant="body" class="text-button text-center">
```